### PR TITLE
Add support to both Legacy and UEFI modes in WIMan templates

### PR DESCRIPTION
**Pull request by amard33p**
Please review @ekohl @UXabre

### CODE REVIEW

File: provisioning_templates/script/windows_default_script.erb [hunk 1]
```diff
@@ -61,7 +61,7 @@ echo Writing install image to partition while downloading additional files
    start /min cmd /C "echo Downloading finsh script and activating SetupComplete.cmd
                 md c:\deploy
                 %WGET% --no-verbose <%= foreman_url("finish") -%> -O C:\deploy\foreman-finish.bat"
-) | pause
+) | pause >nul
```
Comment (amard33p) [inline provisioning_templates/script/windows_default_script.erb:64]: Suppress the 'Press any key to continue' message
foreman_win_fix_script

**Comment by UXabre** [comment]
Hi @amard33p,


Thank you for your contribution, looks good to me and looks like an added value as many people requested this so far! Great work! I haven't been able to test this (corona took my lab away); but I trust you did sufficient tests?


Kind regards,
Arend

**Comment by ekohl** [comment]
I don't have any Windows knowledge in this area, but if @UXabre approves the PR I'm happy to merge it.

**Comment by amard33p** [comment]
@UXabre Yes I have verified these on both VMs and a Dell PE server.

**Review by lzap — APPROVED** [review]
Your words have been put into github records :-)

**Comment by tbrisker** [comment]
On behalf of the Foreman team, thank you for your contribution to the Foreman community templates repository! 
This repository is being archived and all templates are migrated to the relevant repositories. If you wish to pursue this change, please re-open the PR against the relevant repository. For more information and background, please [click here]( https://community.theforeman.org/t/heads-up-community-templates-repo-will-be-archived-tomorrow/19539).

**Pull request closed without merging**